cmd/preview/list_test.go

```diff
@@ -0,0 +1,130 @@
+package preview
+
+import (
+	"context"
+	"fmt"
+	"github.com/okteto/okteto/internal/test/client"
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/types"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_listPreview(t *testing.T) {
+	ctx := context.Background()
+	opts := ListFlags{}
+	var tests = []struct {
+		name           string
+		previews       []types.Preview
+		expectedResult []PreviewOutput
+		expectedErr    error
+	}{
+		{
+			name: "List all previews",
+			previews: []types.Preview{
+				{
+					ID: "test",
+				},
+				{
+					ID: "test-1",
+				},
+			},
+			expectedResult: []PreviewOutput{
+				{
+					Name:     "test",
+					Scope:    "",
+					Sleeping: false,
+					Labels:   "-",
+				},
+				{
+					Name:     "test-1",
+					Scope:    "",
+					Sleeping: false,
+					Labels:   "-",
```

Comment:
we need to add more info to the preview. At least enough to check that the conversion to `previewOutput` works correctly. With the current information we were not catching the bug that we always set to `-` the label field

Reply:
Resolved